### ISSUE TITLE
4.1.3 Status messages and native applications

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#1654]
>  4.1.3 Status Messages (Level AA): **In content implemented using markup languages**, status messages can be programmatically determined through role or properties such that they can be presented to the user by assistive technologies without receiving focus.

that first sentence "In content implemented using markup languages" in practice exempts native applications. I'm not sure this was actually intended? Particularly looking at, say, native apps that show toasts/notifications, this means generally that there's no SC that would apply if those notifications are not announced at all?

**Comment by EmanuelaGorla** [comment]
I agree. That means it is also not applicable to inline error messages displayed in native apps, which are often not announced by screen readers when appearing on screen, resulting in screen reader users not knowing that forms contain errors.

**Comment by bruce-usab** [comment]
At the time, the restriction to content implemented using markup language was quite deliberative.  We didn't think we had standing to apply this SC to things like Flash and Java.  That was well before anyone was thinking about mobile OS native apps, or before any had the idea to apply WCAG to desktop software.

**Comment by patrickhlauke** [comment]
> That was well before anyone was thinking about mobile OS native apps, or before any had the idea to apply WCAG to desktop software.

the heady days of 2018 :)

**Comment by mraccess77** [comment]
It was deliberate - being as this was from 2018 folks were thinking about mobile app, however WCAG is aimed at web content and folks likely didn't want this applied to content like PDF where this might not be supported.  So adding that was the easiest solution to exclude web content where there was not a mechanism.

**Comment by patrickhlauke** [comment]
Suggest that for "WCAG 2.1 to ICT" this exclusion/scoping should probably be lifted, if it hasn't already in whatever draft form it may be in at this point

**Comment by alastc** [comment]
We're gathering people to work on wcag2ICT now, please email me if you're interested.
